### ISSUE TITLE
[@vue/compiler-sfc] <script> and <script setup> must have the same language type.

### ISSUE THREAD

**Issue by Lupeiwen0** [sxzz/unplugin-vue-define-options#20]
When the project uses JS, this error occurs when using defineptions (). 

`[vite] Internal server error:  [@vue/compiler-sfc] <script> and <script setup> must have the same language type.`

Can this plug-in only be used in TS project ?

**Comment by sxzz** [comment]
Could you please provide reproduction please?

**Comment by sxzz** [comment]
This error does not seem to be reported by this plugin. `script` and `script setup` must have the same `lang`.

**Comment by Lupeiwen0** [comment]
Use the `pnpm create vite` instruction to create the project of vite / Vue template
Then execute `pnpm install unplugin-vue-define-options -D` to install unplugin Vue define options

error message: 
```
[vite] hmr update /src/App.vue
[vite] Internal server error: [@vue/compiler-sfc] <script> and <script setup> must have the same language type.
```

vite.config.js ：
```javascript
import { defineConfig } from "vite";
import vue from "@vitejs/plugin-vue";
import DefineOptions from "unplugin-vue-define-options/vite";

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [vue(), DefineOptions()],
});

```

App.vue

```vue
<script setup>
// This starter template is using Vue 3 <script setup> SFCs
// Check out https://vuejs.org/api/sfc-script-setup.html#script-setup
import HelloWorld from './components/HelloWorld.vue'

defineOptions({
  name: 'd-view',
  inheritAttrs: false,
})
</script>

<template>
  <img alt="Vue logo" src="./assets/logo.png" />
  <HelloWorld msg="Hello Vue 3 + Vite" />
</template>

<style>
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
```

**Comment by sxzz** [comment]
Fixed, try `v0.6.1`
